mkt/websites/views.py

```diff
@@ -48,3 +53,11 @@ def as_view(cls, **kwargs):
         # to be wrapped in transactions.
         view = super(WebsiteSearchView, cls).as_view(**kwargs)
         return non_atomic_requests(view)
+
+
+class ReviewersWebsiteSearchView(WebsiteSearchView):
+    permission_classes = [GroupPermission('Apps', 'Review')]
```

Comment:
If this was used within a reviewer tool for website, Websites:Review is fine.  Though as they don't exist currently I'm less sure what it should be (Apps:Review is wrong anyway)